hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/StorageAccess.java

```diff
@@ -30,7 +30,7 @@
  *
  * @param key the key of the access
  * @param value the value read or overwritten
- * @param writtenValue if not Bytes.EMPTY, the overwriting value
+ * @param writtenValue if not null, the overwriting value
```

Comment:
undoing this change from the previous pr as the `writtenValue` _can_ be null if the `StorageAccess` was for a read.